tests/acceptance/common.py

```diff
@@ -299,7 +299,76 @@ def boot_from_internal():
         append("/uboot/uEnv.txt", bootline)
         reboot()
 
-@pytest.fixture(scope="session")
+@pytest.fixture(scope="module")
+def setup_board(request, clean_image, bitbake_variables):
+    bt = pytest.config.getoption("--board-type")
+
+    print('board type:', bt)
+    if bt == "qemu":
+        return qemu_running(request, clean_image)
+    elif bt == "bbb":
+        return setup_bbb(request)
+    elif bt == "colibri-imx7":
+        return setup_colibri_imx7(request, clean_image)
+    else:
+        pytest.skip('board type not configured')
```

Comment:
I would use `fail` here. If board type is somehow misconfigured in Jenkins, I'd rather know about it than get a silent skip. If the test is deselected I don't think the fixture will run anyway, so it should not be a problem for local testing.

Reply:
fixed